cmd/repl/repl_test.go

```diff
@@ -26,12 +27,164 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
-	prompt "github.com/c-bata/go-prompt"
+	"github.com/c-bata/go-prompt"
+	"sqlflow.org/sqlflow/pkg/database"
+	"sqlflow.org/sqlflow/pkg/sql"
+	"sqlflow.org/sqlflow/pkg/sql/testdata"
 )
 
-// TODO(shendiaomo): end to end tests like sqlflowserver/main_test.go
-
 var space = regexp.MustCompile(`\s+`)
+var dbConnStr = "mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0"
+var testDBDriver = os.Getenv("SQLFLOW_TEST_DB")
+var session = makeSessionFromEnv()
+
+func prepareTestData(t *testing.T) error {
+	assertConnectable(dbConnStr)
+	testDB, _ := database.OpenAndConnectDB(dbConnStr)
+	if testDBDriver == "mysql" {
+		_, e := testDB.Exec("CREATE DATABASE IF NOT EXISTS sqlflow_models;")
+		if e != nil {
+			return e
+		}
+		return testdata.Popularize(testDB.DB, testdata.IrisSQL)
+	}
+	t.Skip("Skipping mysql tests")
```

Comment:
Skipping tests that not using mysql

Reply:
I think it means ''We are skipping mysql tests", the same as https://github.com/sql-machine-learning/sqlflow/blob/b4a186551ff0877c534e9a84d3a51673389e0860/cmd/sqlflowserver/main_test.go#L260-L262